src/unix/tcp.c

```diff
@@ -279,44 +279,28 @@ int uv_tcp_open(uv_tcp_t* handle, uv_os_sock_t sock) {
 int uv_tcp_getsockname(const uv_tcp_t* handle,
                        struct sockaddr* name,
                        int* namelen) {
-  socklen_t socklen;
 
   if (handle->delayed_error)
     return handle->delayed_error;
 
-  if (uv__stream_fd(handle) < 0)
```

Comment:
is it OK to remove this?

Reply:
I think so. It should be dealt by [this call](https://github.com/santigimeno/libuv/blob/f8fd063f6849a818863becaf0e63e02799dd0253/src/unix/core.c#L1358) tu `uv_fileno()`